docs/docs/clients/01-typescript.md

```diff
@@ -1,5 +1,4 @@
 ---
-sidebar_position: 1
 description: Information about the generated TypeScript client code.
```

Comment:
So we do not need to include this in the docs?

If we number the titles of the `md` files, do they self-order with docusaurus?